mslib/mswms/mpl_hsec.py

```diff
@@ -63,6 +65,19 @@ def plot_hsection(self):
         """
         pass
 
+    def add_colorbar(self, contour, label=None, tick_levels=None, width="3%", height="30%", format=None):
```

Comment:
I think there were a couple of plots with deviating parameters.
Please add optional parameters for the ""magic numbers" and "magic strings". (i.e. fraction, pad, shrink, loc, extend, ticks_position)
E.g. to allow color bar placement on the left, etc.
Defaults should be the current values.

Reply:
Done